reference/metadata_tables.rst

```diff
@@ -208,23 +210,24 @@ The pg_dist_node table contains information about the worker nodes in the cluste
           3 |       3 | localhost |    12347 | default  | f
     (3 rows)
 
+.. _colocation_group_table:
 
-Colocation group table
+Co-location group table
 ---------------------------------------
 
-The pg_dist_colocation table contains information about which tables' shards should be placed together, or *colocated*. When two tables are in the same colocation group, Citus ensures shards with the same partition values will be placed on the same worker nodes. This enables join optimizations, certain distributed rollups, and foreign key support. Shard colocation is inferred when the shard counts, replication factors, and partition column types all match between two tables; however, a custom colocation group may be specified when creating a distributed table, if so desired.
+The pg_dist_colocation table contains information about which tables' shards should be placed together, or *co-located*. When two tables are in the same co-location group, Citus ensures shards with the same partition values will be placed on the same worker nodes. This enables join optimizations, certain distributed rollups, and foreign key support. Shard co-location is inferred when the shard counts, replication factors, and partition column types all match between two tables; however, a custom co-location group may be specified when creating a distributed table, if so desired.
```

Comment:
Maybe add another link here to our high-level description section on co-location?